lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
```

Comment:
It's fine to put the "For example" on the same line, we do that in many other situations. Git blame for documentation is a bit more loose since changes to the documentation aren't blamed as often as code IMO.